### ISSUE TITLE
Request: Add `permission_classes` to responses

### ISSUE THREAD

**Issue by jvacek** [tfranzel/drf-spectacular#1152]
I've got a couple `permissions.BasePermission` classes that have the `message` field set on them.

```py
class MyAccessPermissions(permissions.BasePermission):
    """Checks if the instance is accessible"""

    message = "instance does not own a subscription"

    def has_permission(self, request, view) -> bool:
        #code
```
It would be nice if these could be included under `403` in the `responses` list, with the messages in the examples.

**Comment by tfranzel** [comment]
Due to the inherent complexities and generic nature of DRF error handling, we cannot provide that feature in a reasonable manner. If you want to have that information in the schema, if you need to use `@extend_schema`.

**Comment by jvacek** [comment]
Is there a way that I can annotate my permission classes directly at least, without having to annotate all the views that use them? I did not find any such mention in the documentation.

**Comment by tfranzel** [comment]
No, because permission classes would be only a part of the solution.

You can however, save the common stuff and e.g. reuse the list of responses for example. 

People also have customized the AutoSchema: https://github.com/tfranzel/drf-spectacular/issues/101#issuecomment-1069383391
